File: app/src/main/java/vn/loitp/app/activity/customviews/layout/coordinatorlayout/CoordinatorLayoutActivity.java
package vn.loitp.app.activity.customviews.layout.coordinatorlayout;

import android.os.Bundle;

import loitp.basemaster.R;
import vn.loitp.core.base.BaseFontActivity;

public class CoordinatorLayoutActivity extends BaseFontActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

    }

    @Override
    protected boolean setFullScreen() {
        return false;
    }

    @Override
    protected String setTag() {
        return getClass().getSimpleName();
    }

    @Override
    protected int setLayoutResourceId() {
        return R.layout.activity_coordinator_layout;
    }
}

File: app/src/main/java/vn/loitp/app/activity/customviews/bottomBar/expandableBottomBar/screens/CoordinatorLayoutActivity.kt
package vn.loitp.app.activity.customviews.bottomBar.expandableBottomBar.screens

import android.os.Bundle
import com.google.android.material.snackbar.Snackbar
import com.loitp.annotation.IsAutoAnimation
import com.loitp.annotation.IsFullScreen
import com.loitp.annotation.LogTag
import com.loitpcore.core.base.BaseFontActivity
import kotlinx.android.synthetic.main.activity_ebb_coordinator_layout.*
import vn.loitp.app.R

@LogTag("CoordinatorLayoutActivity")
@IsFullScreen(false)
@IsAutoAnimation(false)
class CoordinatorLayoutActivity : BaseFontActivity() {

    override fun setLayoutResourceId(): Int {
        return R.layout.activity_ebb_coordinator_layout
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)


        fab.setOnClickListener { view ->
            Snackbar.make(view, "Meow", Snackbar.LENGTH_LONG).show()
        }
    }

}
